src/rules/lambda/LambdaLogging.ts

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+import { parse } from 'path';
+import { CfnResource, Stack } from 'aws-cdk-lib';
+import { CfnFunction } from 'aws-cdk-lib/aws-lambda';
+import { NagRuleCompliance } from '../../nag-rules';
+
+/**
+ * Ensure that Lambda functions have a corresponding Log Group
```

Comment:
```suggestion
 * Lambda functions explicitly define their CloudWatch Log Groups
```

 Rule comments are declarative